esp/services/ldapenvironment/ldapenvironmentService.cpp

```diff
@@ -238,11 +238,11 @@ bool CldapenvironmentEx::onLDAPCreateEnvironment(IEspContext &context, IEspLDAPC
         if (req.getCreateLDAPEnvironment())
         {
             //Note that ESP will also try to create these OUs on startup
-            createLDAPBaseDN(respFilesBaseDN.str(), description.str(), notes);
-            createLDAPBaseDN(respGroupsBaseDN.str(), description.str(), notes);
-            createLDAPBaseDN(respUsersBaseDN.str(), description.str(), notes);
-            createLDAPBaseDN(respResourcesBaseDN.str(), description.str(), notes);
-            createLDAPBaseDN(respWorkunitsBaseDN.str(), description.str(), notes);
+            createLDAPBaseDN(respFilesBaseDN.str(), PT_DEFAULT, description.str(), notes);
+            createLDAPBaseDN(respGroupsBaseDN.str(), PT_ADMINISTRATORS_ONLY, description.str(), notes);
+            createLDAPBaseDN(respUsersBaseDN.str(),  PT_ADMINISTRATORS_ONLY, description.str(), notes);
+            createLDAPBaseDN(respResourcesBaseDN.str(), PT_ADMINISTRATORS_ONLY, description.str(), notes);
+            createLDAPBaseDN(respWorkunitsBaseDN.str(), PT_DEFAULT, description.str(), notes);
```

Comment:
In this PR, The perms for FILES and WU OUs are changed to PT_DEFAULT. Is it possible that some perms have to be changed again or for some special use cases? If yes, should we allow that some perms can be set from the req?

Reply:
These perms settings have always been hardcoded in ldapconnection.cpp CLdapClient::init(), and are applied when the security manager starts.  This code mimicks that behavior, by pre-creating the OU with the same permissions as is applied in CLdapClient::init.  I don't see when this would ever change, except perhaps if new LDAP permission policy , in which a code change would be required.  I don't see a reason to make them dynamic